neo/rawio/nixrawio.py

```diff
@@ -324,3 +355,24 @@ def _rescale_epoch_duration(self, raw_duration, dtype='float64'):
         durations = raw_duration.astype(dtype)
         # supposing unit is second, other possibilities maybe mS microS...
         return durations  # return in seconds
+
+    def _add_annotate(self, tar_ann, props, otype):
```

Comment:
This function could be more general if you would use Neo object types as `otype`.